Translation for the page 'API Reference -> SyntheticEvent' (#140)
reactjs/it.reactjs.org: content/docs/reference-events.md

}
```

> Nota:
>
> Se vuoi avere accesso le proprietà in un modo asincrono, dovresti invocare `event.persist()` sull'evento, quale rimuoverà l'evento sintetico dal pool e permettono riferimenti all'evento essere conservato da codice utente.

## Eventi Supportati {#supported-events}

React normalizza eventi così che hanno proprietà coerente in tutti browser.

L'event handlers sotto sono scatenato da un event in la fase di gorgogliante. Per registrare un event handler per la fase di salvataggio, affiggi `Capture` al nome dell'evento; per esempio, invece di usando `onClick`, useresti `onClickCapture` per gestire l'evento della clicca nella fase salvataggia.

- [Eventi degli Appunti](#clipboard-events)
- [Eventi della Composizione](#composition-events)
- [Eventi della Tastiera](#keyboard-events)
- [Eventi di Focus](#focus-events)
- [Eventi di Form](#form-events)
- [Eventi del Mouse](#mouse-events)

> - [Eventi del Cursore](#pointer-events)

Reviewer: ```suggestion
- [Eventi del Puntatore](#pointer-events)
```

- [Eventi della Selezione](#selection-events)

> - [Eventi Tattile](#touch-events)

Reviewer: same correction below on its target, please

> - [Eventi della Interfaccia Utente](#ui-events)

Reviewer: same correction below on its target, please

> - [Eventi della Ruota](#wheel-events)

Reviewer: same correction below on its target, please

> - [Eventi degli Media](#media-events)

Reviewer: same correction below on its target, please

> - [Eventi della Immagine](#image-events)

Reviewer: ```suggestion
- [Eventi dell'Immagine](#image-events)
```

Reviewer: same correction below on its target, please

number which
```

> La `key` proprietà può accettare uno qualsiasi dei valori documentati nel [DOM Livello 3 Eventi specificazione](https://www.w3.org/TR/uievents-key/#named-key-attribute-values).

Reviewer: ```suggestion
La proprietà `key` può accettare uno qualsiasi dei valori documentati nelle [specifiche degli Eventi del DOM Livello 3](https://www.w3.org/TR/uievents-key/#named-key-attribute-values).
```

* * *

### Eventi di Focus {#focus-events}

Nomi dell'eventi:

```
onFocus onBlur
```

> Questi eventi di focus lavorano su tutti elementi nel React Dom, non solo elementi di form.

Reviewer: ```suggestion
Questi eventi di focus funzionano con tutti elementi nel React DOM, non solo elementi form.
```

Proprietà:

```javascript
DOMEventTarget relatedTarget
```

* * *

### Eventi di Form {#form-events}

Nomi dell'eventi:

```
onChange onInput onInvalid onSubmit
```

> Per più informazioni For more information sul evento onChange, vedi [Forms](/docs/forms.html).

Reviewer: ```suggestion
Per maggiori informazioni sull'evento onChange, vedi [Forms](/docs/forms.html).
```

* * *

### Eventi del Mouse {#mouse-events}

Nomi dell'eventi:

```
onClick onContextMenu onDoubleClick onDrag onDragEnd onDragEnter onDragExit
onDragLeave onDragOver onDragStart onDrop onMouseDown onMouseEnter onMouseLeave
onMouseMove onMouseOut onMouseOver onMouseUp
```

> Il `onMouseEnter` e `onMouseLeave` eventi si propagano dal elemento essendo lasciato al uno essendo entrato invece il gorgogliante ordinario e non hanno una fase salvataggia.

Reviewer: ```suggestion
Gli eventi `onMouseEnter` e `onMouseLeave` vengono propagati dall'elemento che viene lasciato a quello che viene acceduto invece di seguire il normale _bubbling_ e non hanno una fase di _capture_. (Clicca [qui](https://developer.mozilla.org/it/docs/Learn/JavaScript/Building_blocks/Eventi#Event_bubbling_and_capture) per maggiori informazioni su _bubbling_ e _capture_.
```

* * *

> ### Eventi del Cursore {#pointer-events}

Reviewer: ```suggestion
### Eventi del Puntatore {#pointer-events}
```

Nomi dell'eventi:

```
onPointerDown onPointerMove onPointerUp onPointerCancel onGotPointerCapture
onLostPointerCapture onPointerEnter onPointerLeave onPointerOver onPointerOut
```

> Il `onPointerEnter` e `onPointerLeave` eventi si propagano dal elemento essendo lasciato al uno essendo entrato invece il gorgogliante ordinario e non hanno una fase salvataggia.

Reviewer: ```suggestion
Gli eventi `onPointerEnter` e `onPointerLeave` vengono propagati dall'elemento che viene lasciato a quello che viene acceduto invece di seguire il normale _bubbling_ e non hanno una fase di _capture_. (Clicca [qui](https://developer.mozilla.org/it/docs/Learn/JavaScript/Building_blocks/Eventi#Event_bubbling_and_capture) per maggiori informazioni su _bubbling_ e _capture_.
```

Proprietà:

> Come definato nel [W3 specificazione](https://www.w3.org/TR/pointerevents/), eventi del cursore estendono [Eventi del Mouse](#mouse-events) con le seguente proprietà:

Reviewer: ```suggestion
Come definito nelle [specifiche W3](https://www.w3.org/TR/pointerevents/), gli eventi del cursore estendono gli [Eventi del Mouse](#mouse-events) con le seguente proprietà:
```

boolean isPrimary
```

> Una nota di cross-browser supporto:

Reviewer: ```suggestion
Una nota sul supporto cross-browser:
```

> Eventi del cursore non sono ancora supportato in tutti browser (al tempo di scrittando questo articolo), browsers supportati includono: Chrome, Firefox, Edge, e Internet Explorer). React deliberatamente non polyfill supporto per altri browser perché un conforme-normale polyfill aumenterebbe il dimensione di carico di `react-dom` molto.

Reviewer: ```suggestion
Gli eventi del puntatore non sono ancora supportati in tutti i browser (al momento della scrittura di questo articolo), tra quelli supportati abbiamo: Chrome, Firefox, Edge, e Internet Explorer). React deliberatamente non offre supporto agli altri browsers mediante polyfill in quanto ciò aumenterebbe in modo considerevole la dimensione del pacchetto `react-dom`.
```

> Se tua applicazione desideri eventi del cursore, raccomandiamo aggiungiendo un cursore evento polyfill di terzo.

Reviewer: ```suggestion
Se la tua applicazione richiede l'utilizzo degli eventi del puntatore, raccomandiamo l'uso di una polyfill specifica di terze parti.
```
